app/Footer.tsx

```diff
@@ -36,9 +36,6 @@ export default function Footer() {
             Beam status
           </Link>
         </nav>
-        <p className="mt-8 text-base leading-6 text-center text-gray-400">
-          © 2024 Experiment Controls at ISIS. All rights reserved.
-        </p>
       </div>
```

Comment:
The footer is now quite big but quite empty vertically

Maybe either make footer smaller vertically - or do we actually _need_ a footer any more?

Reply:
we need it for mobile displays I think as they don't have a navbar, maybe they should have an expandable one

Reply:
that being said we probably don't need another ibex logo, will remove